draft-ietf-mls-protocol.md

```diff
@@ -1150,9 +1157,8 @@ struct {
 } optional<T>;
 
 struct {
-    HPKEPublicKey public_key;
+    ParentHashInput parent_hash_input;
```

Comment:
I prefer not to group things like this, and just assemble the hash input as necessary.  Especially since #436 is going to change `ParentHashInput` to diverge further from `ParentNode`.  Implementations are of course free to store things this way.